games-action/cro_mag_rally/cro_mag_rally-3.0.1.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Racing game set in prehistoric times"
+DESCRIPTION="In Cro-Mag Rally you are a speed-hungry caveman named Brog who races through the Stone, Bronze, and Iron Ages in primitive vehicles such as the Geode Cruiser, Bone Buggy, Logmobile, Trojan Horse, and many others. Brog has at his disposal an arsenal of primitive weaponry ranging from Bone Bombs to Chinese Bottle Rockets and Heat Seeking Homing Pigeons.
+In addition to single-player racing where one player races against the computer, there are also several different multi-player modes including Tag, Capture the Flag, and Survival. Up to four players can play on a single computer in split-screen mode"
+HOMEPAGE="https://github.com/jorio/CroMagRally"
+COPYRIGHT="2000 Pangea Software, Inc."
+LICENSE="CC-BY-NC-SA-4.0"
+REVISION="1"
+SOURCE_URI_1="https://github.com/jorio/CroMagRally/archive/refs/tags/3.0.1.zip"
+CHECKSUM_SHA256_1="6e31945791ab19169a41667f6a1a3802c10b30e435079776c440518c1d34c505"
+SOURCE_DIR_1="CroMagRally-$portVersion"
+PATCHES="cro_mag_rally-3.0.1.patchset"
+srcGitRev2="ef94150e2dcec522e3099f4d03a4e8f2639f7232"
+SOURCE_URI_2="https://github.com/jorio/Pomme/archive/$srcGitRev2.tar.gz"
+CHECKSUM_SHA256_2="69faf616449c0737e238367883e36230f15d13c261e18d6e4d4c69e2edd0b002"
+SOURCE_FILENAME_2="Pomme-$srcGitRev2.tar.gz"
+SOURCE_DIR_2="Pomme-$srcGitRev2"
+PATCHES_2="cro_mag_rally-$portVersion-source2.patchset"
+ADDITIONAL_FILES="cro_mag_rally.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cro_mag_rally$secondaryArchSuffix = $portVersion
+	app:CroMagRally = $portVersion
+	"
+REQUIRES="
+        haiku
+        lib:libgl$secondaryArchSuffix
```

Comment:
Nitpicking :)
```diff
-        lib:libgl$secondaryArchSuffix
+        lib:libGL$secondaryArchSuffix
```
In BUILD_REQUIRES also.

Reply:
Done :)